bindings/rust/bench/README.md

```diff
@@ -25,3 +35,20 @@ All certs are stored in `certs/` and can be regenerated using `certs/generate_ce
 ### Negotiation parameters
 
 The cipher suites benchmarked are `TLS_AES_128_GCM_SHA256` and `TLS_AES_256_GCM_SHA384`, and the key exchange methods benchmarked are ECDHE with `secp256r1` and with `x25519`. We also test connections with and without client authentication (mTLS).
+
+## Sample output
+
+### Historical performance
+
+Because these benches take a longer time to generate (>30 min), we include the results from historical benching here.
+
+Notes: 
+- Two sets of parameters for the handshake couldn't be benched before 1.3.40, since security policies that negotiated those policies as their top choice did not exist before then.
+- There is no data from 1.3.30 to 1.3.37 because those versions have a dependency issue that cause the Rust bindings not to build. However, there is data before and after that period, so the performance for those versions can be inferred via interpolation.
+- The improvement in throughput in 1.3.28 was most likely caused by the addition of LTO to the default Rust bindings build. 
+- Since the benches are run over a long time, noise on the machine can cause variability, as seen in the throughput graph.
+- The variability can be seen with throughput especially because it is calculated as the inverse of time taken.
+
+![historical-perf-handshake](https://github.com/tinzh/s2n-tls/assets/76919968/b6448634-e6d1-4724-ab91-7efc26485274)
```

Comment:
Ok got it. Do you see all of the images, including the ones from memory benching, living here? Or just these particular images because they're in the readme (because they take a while to generate?)

Reply:
I don't think it necessarily makes sense to have standalone images committed into the repo. It seems better if they're included as part of the documentation, so you can give some context for them like you did in this readme. So I think images should just be included that are used in the readme.

I think it also makes sense particularly for the historical benchmarking because the single snapshot is useful. For other benchmarking, showing results from old versions doesn't seem as useful.

Reply:
Ok got it. I'll add *.svg back into the gitignore but just specifically allowlist the historical-perf svgs. Maybe other generated images can still be put into `images/`, but they just won't be checked in.